constraints.md

```diff
@@ -80,11 +70,13 @@ SELECT * FROM users;
 
 ## UNIQUE KEY
 
-Depending on the transaction mode and the value of `tidb_constraint_check_in_place`, TiDB might check `UNIQUE` constraints [lazily](/transaction-overview.md#lazy-check-of-constraints). This helps improve performance by batching network access.
+Unique constraints mean that all non-null values in a unique index and a primary key column are unique.
 
-For example:
+### Optimistic transactions
 
-{{< copyable "sql" >}}
+In optimistic transactions, TiDB checks unique constraints [lazily](/transaction-overview.md#lazy-check-of-constraints) by default. When an optimistic transaction is committed, TiDB performs a bulk check, which helps reduce network overhead and improve performance.
```

Comment:
How about "When an optimistic transaction is committed, TiDB performs a batch check on unique constraints in this transaction, which helps reduce network overhead and improve performance."?

Reply:
Can be "... by default, namely these checks are performed in the commit phase, which ...

Reply:
"namely these checks are performed in the commit phase" might give the confusion that "these checks are not performed in the execution phase"?

Reply:
Emm you are right. Some checks could be performed in the execution phase. How about "some checks may be performed in the commit phase"?

Reply:
It might be hard to know what are "some check may be", how about "By default, for optimistic transactions, TiDB checks unique constraints [lazily](https://github.com/transaction-overview.md#lazy-check-of-constraints) in the execution phase and strictly in the commit phase, which helps reduce network overhead and improve performance."

Reply:
Sounds OK to me

Reply:
Done.